spec/system/planning_applications/assessment_tasks_index_spec.rb

```diff
@@ -8,6 +8,8 @@
 
   before do
     sign_in assessor
+
+    allow(ENV).to receive(:fetch).with("PLANNING_HISTORY_ENABLED", "false").and_return("true")
```

Comment:
In these feature specs it might be worth also testing for the case where the env var is blank or "false".